tools/gyp/.github/workflows/Python_tests.yml

```diff
@@ -11,19 +11,19 @@ jobs:
       max-parallel: 8
       matrix:
         os: [macos-latest, ubuntu-latest] # , windows-latest]
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
Shouldn't this file be upstreamed via https://github.com/nodejs/gyp-next/blob/main/.github/workflows/python_tests.yml?

Reply:
I think we want to do more upstream like:
1. Test pre-release Python (Currently 3.12 alpha 7)
2. Test that an editable test of the app works.
3. Ensure that the app responds to `gyp --version`, etc.

Once the app is promoted here (gyp-next --> node-gyp --> node), these tests can focus more on installing node-gyp with Node.js in all the environments.

Reply:
I mean to ask if these changes should be first performed in the `gyp-next` repo, and only then we should update it?

Reply:
Which changes?

Reply:
Now I get it...  Those changes are reverted.  Thx.